### PR TITLE
Use code owner teams for every component.

### PR DESCRIPTION
**Pull request by Zimmi48**
It was decided during the Coq WG that code owner teams are more convenient, in particular because they allow adding and removing team members without going through a pull request.  For each team, we should aim to have at least three code owners, even if in some cases we are going to start with less.

**Kind:** meta.

Supersedes / closes #10254.

The newly created teams are:

For components:
- @coq/kernel-maintainers (kernel + coqchk)
- @coq/vm-native-maintainers (VM + native compute)
- @coq/lib-maintainers (lib + clib)
- @coq/engine-maintainers (engine + proof)
- @coq/universes-maintainers (universes in kernel and engine)
- @coq/coqide-maintainers
- @coq/stm-maintainers (including XML protocol)
- @coq/extensible-syntax-maintainers (interp + printing)
- @coq/library-maintainers 
- @coq/pretyper-maintainers
- @coq/tactics-maintainers
- @coq/typeclasses-maintainers (typeclasses-related files in various places)
- @coq/toplevel-maintainers (toplevel + topbin)
- @coq/vernac-maintainers
- @coq/stdlib-maintainers

For infrastructure:
- @coq/legacy-build-maintainers (Makefile + configure + coqdep)
- @coq/build-maintainers (Dune + configure + coqdep)
- @coq/windows-build-maintainers
- @coq/compat-maintainers
- @coq/test-suite-maintainers

For tools:
- @coq/coqdoc-maintainers
- @coq/coq-makefile-maintainers
- ~~@coq/coqwc-maintainers~~
- @coq/dev-tools-maintainers

For plugins:
- @coq/btauto-maintainers
- @coq/cc-maintainers 
- @coq/derive-maintainers
- @coq/extraction-maintainers
- @coq/firstorder-maintainers
- @coq/funind-maintainers
- @coq/ltac-maintainers
- @coq/ltac2-maintainers
- @coq/nsatz-maintainers
- @coq/rtauto-maintainers

### CODE REVIEW

**Comment by ejgallego** [comment]
Some comments:
- I think we should add some help on how to actually figure out which groups a particular developer is, IMHO there are quite a lot of groups as to be able to remember,
- I'm not sure I like the fact that maintainers are not formally documented now; I like for example Linux's MAINTAINERS file, the rest of the data could be kept in sync with some script,
- I am not convinced the current review/assign model is scalable, for example when compared with feature / subsystem trees as done in Linux. So while this update could work for now I have a feeling that it should not be done by itself, but in coordination with a revision of the development / contribution policies.

File: .github/CODEOWNERS
Comment (Zimmi48) [inline .github/CODEOWNERS]: Changelog updates do not trigger review request anymore as was requested during the WG.

**Comment by maximedenes** [comment]
So, what should I do? Not merge until we address @ejgallego 's remarks, or merge first?

**Comment by maximedenes** [comment]
@Zimmi48 did you create the teams? I don't see them here: https://github.com/orgs/coq/teams/pushers/teams but maybe I'm looking at the wrong place.

**Comment by ejgallego** [comment]
> So, what should I do? Not merge until we address @ejgallego 's remarks, or merge first?

I don't think my comments are blocking; I wrote them here as I couldn't participate in the discussion in person.

**Comment by maximedenes** [comment]
> I don't think my comments are blocking; I wrote them here as I couldn't participate in the discussion in person.

Ok, thanks.

**Comment by Zimmi48** [comment]
> @Zimmi48 did you create the teams? I don't see them here: https://github.com/orgs/coq/teams/pushers/teams but maybe I'm looking at the wrong place.

No, I was waiting to see if I got comments on the proposed list. I'll start creating the teams but it could take a few more days to populate them all.

**Comment by Zimmi48** [comment]
I realized the merging doc must also be updated to reflect that the team setup is now the default setup.

**Comment by Zimmi48** [comment]
Teams created. Now there will be some more work to populate them.

**Comment by Zimmi48** [comment]
Who wants to be part of the new @coq/build-maintainers team beside Emilio and myself?

**Comment by ejgallego** [comment]
> Who wants to be part of the new @coq/build-maintainers team beside Emilio and myself?

@anton-trunov @SkySkimmer just in case you are interested in getting in the build team, you are most welcome; of course no pressure / you could drop anytime.

**Comment by Zimmi48** [comment]
@coq/pushers As a complement to this PR, I am moving membership from the pushers team directly to relevant sub-teams. It is possible that you receive e-mail notifications about you being removed from the @coq/pushers team, but no one is losing any permission through this process (as people who are concerned by this are already members of some sub-teams).

**Comment by Zimmi48** [comment]
The @coq/coq-makefile-maintainers team is Enrico + Guillaume for now (former primary and secondary owners). However, I don't know if Guillaume is still interested in being part of this team. If there are other volunteers, feel free to let us know.

**Comment by Zimmi48** [comment]
The @coq/coqdoc-maintainers team is Guillaume + Matthieu for now (former primary and secondary owners). I don't know if any of them is still interested in being part of this team. On the other hand, maybe there are some new volunteers to help review and merge coqdoc-related pull requests: @Lysxia, @proux01, @ejgallego, @herbelin, ...?

**Comment by SkySkimmer** [comment]
>@anton-trunov @SkySkimmer just in case you are interested in getting in the build team, you are most welcome; of course no pressure / you could drop anytime.

I am willing

**Comment by Zimmi48** [comment]
The @coq/coqide-maintainers team is Hugo + Pierre-Marie based on current interest. @MSoegtropIMC should I add you to it as well? Any other volunteer?

**Comment by Zimmi48** [comment]
The @coq/coqwc-maintainers is only Guillaume for now. We would need at least one additional volunteer, preferably two or more.

**Comment by gares** [comment]
didn't we put coqdoc and coqwc in the same pot?

**Comment by ejgallego** [comment]
> @coq/coqide-maintainers team is Hugo + Pierre-Marie based on current interest.

I can help a bit with coqide, given that I've been involved in lablgtk3 would make sense.

I was going to list the teams I could be involved with, but I realized @Zimmi48 this will be a bit messy for everyone, why not create a doodle where columns are teams, rows are people, and they have "interested" "a bit, only if no enough members", "not" ?

**Comment by Zimmi48** [comment]
> didn't we put coqdoc and coqwc in the same pot?

That was proposed, but we didn't because some volunteer to maintain coqdoc might have no idea about coqwc...

**Comment by gares** [comment]
> maintain coqdoc might have no idea about coqwc

Yes this was you argument, here mine: if you are in group X it does not mean you have to be competent or willing to maintain the entire code assigned to X.

A loose definition of maintainer can scary people, not a 300 lines parser essentially counting how many times Definition or Proof occurs in a text file.

Also the only maintainer in coqwc is also in coqdoc, so if you notify coqdoc you reach him.

**Comment by anton-trunov** [comment]
> @anton-trunov @SkySkimmer just in case you are interested in getting in the build team, you are most welcome; of course no pressure / you could drop anytime.

Thank you, folks. I'm in :)

**Comment by Zimmi48** [comment]
@gares OK, thank you for the additional explanation. I guess this is up for discussion. Will be easy to change before the PR is completed. I am not opposed.

**Comment by Zimmi48** [comment]
>I was going to list the teams I could be involved with, but I realized @Zimmi48 this will be a bit messy for everyone, why not create a doodle where columns are teams, rows are people, and they have "interested" "a bit, only if no enough members", "not" ?

I've created such a poll. Let's see what we can do with it: https://framadate.org/IMqhZndDG7prOO8I
The reference list for the teams (should be readable to anyone in the Coq org) is: https://github.com/orgs/coq/teams/pushers/teams.

**Comment by Zimmi48** [comment]
@Lysxia As a new volunteer code owner for coqdoc, what do you think of Enrico's proposal to include `coqwc` in the scope of the team?

**Comment by Zimmi48** [comment]
@ppedrot @herbelin @mattam82 @silene It would be great if you could fill the poll at https://framadate.org/IMqhZndDG7prOO8I to share your current interests / committments. Other participants are welcome to fill it too! I'm just sending a reminder to the core developers who have not filled it yet.

**Comment by Zimmi48** [comment]
The @coq/extraction-maintainers team needs more members! @pi8027: Would you be interested in joining this team? (This is about reviewing and merging incoming PRs on extraction by other contributors.)

**Comment by Lysxia** [comment]
> @Lysxia As a new volunteer code owner for coqdoc, what do you think of Enrico's proposal to include `coqwc` in the scope of the team?

I'm fine with that plan! (I don't mind it either way, and I could imagine myself giving a hand once in a while.)

**Comment by Zimmi48** [comment]
The @coq/ltac2-maintainers needs members beside @ppedrot! @kyoDralliam: You said you would be potentially interested? @MSoegtropIMC: What about you?

**Comment by Zimmi48** [comment]
@coq/reals-library-maintainers: This could be an occasion to expand the team. For instance, @VincentSe and @spitters might be interested in joining it...

**Comment by MSoegtropIMC** [comment]
@Zimmi48 : yes please add me to the CoqIDE maintainers (I will take care of Windows specific issues as in the past) and also to Ltac2 where I will look into libraries and usability. In case it exists I might want to join the OSX build mainatiners as well some time soon.

**Comment by Zimmi48** [comment]
> In case it exists I might want to join the OSX build mainatiners as well some time soon.

It doesn't exist yet. We could create this team and it would be the natural owner of the (small) file under `dev/build/osx`. However, a large portion of the macOS build logic is actually contained in the `azure-pipelines.yml` file, which is in the realm of the CI maintainers team.

**Comment by pi8027** [comment]
> The @coq/extraction-maintainers team needs more members! @pi8027: Would you be interested in joining this team? (This is about reviewing and merging incoming PRs on extraction by other contributors.)

@Zimmi48 Yes, please add me. I would like to learn how to work as a maintainer. :)

**Comment by herbelin** [comment]
@zimmi48: I filled the poll. (Sorry, I did not realize that this thread was concerning everyone and I had skipped it until today.) Basically, in addition to topics I'm particularly familiar with (such as notations, elaboration, tactics, ...), my rough position is to help where noone else can help.

**Comment by Zimmi48** [comment]
@herbelin: Did you actually submit your answer? I can't see it.

**Comment by Zimmi48** [comment]
I have updated the merging documentation to reflect the changes in the process (only maintainer teams now) and took the opportunity to simplify it and integrate it in the contributing guide. This was the last blocker and it should now be possible to merge this PR (and we can continue updating the code owner teams based on answers to the poll even after the PR itself is merged).


@anton-trunov @Lysxia @pi8027: you are new maintainers, so please make sure you read https://github.com/coq/coq/blob/8fd24bc02e5ed03593ee665d9a52903f97b8eac5/CONTRIBUTING.md#becoming-a-maintainer.

**Comment by Zimmi48** [comment]
@tlringer: I realize that you are not already part of the @coq/plugin-tutorial-maintainers team. Given your track record of contributing to improve the plugin tutorial, you would be very much welcome to join this team (i.e. participate to review and merge PRs from other authors on the plugin tutorial) if you want to.

**Comment by Zimmi48** [comment]
@maximedenes: This is ready to merge whenever you are.

@herbelin: I still don't see your answer to the poll.

@mattam82 @silene @vbgl: It would be great if you could fill it when you get the chance. It shouldn't take long and will avoid keeping you in teams where you don't plan on helping.

Link: https://framadate.org/IMqhZndDG7prOO8I

**Review by maximedenes — APPROVED** [review]

**Comment by Zimmi48** [comment]
FTR I've also invited @VincentSe to join @coq/reals-library-maintainers since he answered positively to the proposal.

**Comment by Zimmi48** [comment]
Given that there is no Coq call scheduled today, I'll write about the things I wanted to say / ask, and maybe we can talk more about it next week.

It seems to me that the suggestion that was made during the last WG to go to full code owner teams was a good choice. It allowed us to realize that in many areas, there are more than just two volunteers willing to review incoming PRs (sometimes additional volunteers could be found among core developers, sometimes among regular contributors). We have jumped from 25 to 29 members of the "pushers" team (now there are no direct members of this team, only members of sub-teams of the "pushers" team).

Furthermore, this realization was emphasized thanks to the idea of creating a poll (that was suggested here by @ejgallego). Unfortunately, as this happened during the Christmas break, some people have yet to answer (in particular @mattam82, @silene and @vbgl among core developers). Nonetheless, it seems that this idea was successful and I think it could make sense to re-run such a poll every year, and that this would give the whole development team a better view of what commitments the various developers are willing to take.

Speaking of giving a better view of the development team, here is a wiki page that I created during the last WG, and to which I added today the list of all maintainers: https://github.com/coq/coq/wiki/Coq-development-team. It would be great if everyone could complete their profile with up-to-date info. In my own profile, I've added a new line "*currently a maintainer of*". It would be great if everyone could add this line as well so that this information can be found by people who are not part of the organization (https://github.com/orgs/coq/teams/pushers/teams is not visible to non-members).

Finally, the poll proposed to answer not just "yes" or "no" but also "maybe" which corresponds to "only if there are not enough volunteers". There remains to determine what "enough volunteers" mean. The following would likely have been simpler to do during the call:

- @coq/btauto-maintainers is @ppedrot (yes) and @herbelin (maybe, previous code owner).
- @coq/cc-maintainers is @herbelin (yes) and @PierreCorbineau (previous code owner): should we add @ppedrot (maybe) to get to a team of three?
- @coq/coqide-maintainers has already three "yes", so not adding the one "maybe" (@gares).
- @coq/derive-maintainers is @aspiwack (previous primary code owner), @herbelin (yes), and @ppedrot (previous secondary code owner, maybe).
- @coq/doc-maintainers has already three people, but @cpitclaudel does not have so much time available, so it could make sense to have one more person join the team (the people that answered "maybe" are @anton-trunov and @herbelin).
- @coq/engine-maintainers has three "yes" and three "maybe". I guess the three "yes" should be enough.
- @coq/extensible-syntax-maintainers is @herbelin, @ejgallego and @mattam82 (previous code owner). It has one additional "maybe" (@ppedrot) if needed.
- @coq/extraction-maintainers has only two members (@pi8027 and @maximedenes). Should we add one of the persons that answered maybe (@ejgallego, @herbelin)?
- @coq/funind-maintainers has only two members (@Matafou, @forestjulien). Should we add new members among the ones that answered "maybe" (@ejgallego, @herbelin)?
- @coq/kernel-maintainers has three "yes" and three "maybe". I guess the three "yes" should be enough.
- @coq/lib-maintainers has only one "yes" but three "maybe". Who should we add to the team among them (@SkySkimmer, @ejgallego, @herbelin)?
- @coq/library-maintainers has only two members: @maximedenes (yes) and @silene (previous code owner). Who should we add to the team among the four "maybe" (@SkySkimmer, @ejgallego, @ppedrot, @herbelin)?
- @coq/ltac-maintainers has only two members (@herbelin, @ppedrot). Is a third one needed (@JasonGross answered "maybe")?
- @coq/ltac2-maintainers has only two members (@ppedrot, @MSoegtropIMC). Should we add someone from those who answered "maybe" (@herbelin, @JasonGross)?
- @coq/micromega-maintainers already had four members so probably not needed to add anyone.
- @coq/nsatz-maintainers has three members: @herbelin (yes), @ppedrot (previous code owner, maybe), @thery (previous code owner).
- @coq/plugin-tutorial-maintainers has plenty of members but only received "maybe" in the poll. So far, I only removed @ppedrot from the team because he answered "no".
- @coq/pretyper-maintainers has four members (three "yes" + @mattam82, previous code owner). So probably not needed to add anyone to this team.
- @coq/stdlib-maintainers has only two members (@herbelin and @anton-trunov). Should one of the "maybe" join (@Zimmi48, @JasonGross)?
- @coq/stm-maintainers has only one member (@gares). Should we add the two that answered "maybe" (@maximedenes, @herbelin)?
- @coq/tactics-maintainers has three members (@herbelin, @ppedrot, @mattam82) so might not need any of the additional three "maybe".
- @coq/test-suite-maintainers received only one "yes" (@SkySkimmer). Should we add the two that answered "maybe" (@ejgallego, @herbelin)?
- @coq/toplevel-maintainers only has two members that answered "yes" (@gares, @ejgallego). Should they be joined by the only "maybe" (@herbelin)?
- @coq/typeclasses-maintainers has two members (@ppedrot and @mattam82, previous code owner). @ppedrot you did not answer "yes" nor "maybe" here, so do you wish to leave the team? Should we add @herbelin who answered "maybe" to it in any case?
- @coq/universes-maintainers has only two members (@mattam82, @SkySkimmer). Does it need more members (@ppedrot and @herbelin answered "maybe")?
- @coq/vernac-maintainers has already three members (@mattam82, @maximedenes, @ejgallego) so might not need any of the two who answered "maybe".
- @coq/vm-native-maintainers has just three members: @maximedenes (yes), @ppedrot (maybe), and @silene (previous code owner).
- @coq/windows-build-maintainers has two members: @MSoegtropIMC and myself (maybe). Should we add any of the two other "maybe" (@gares, @herbelin)?

Note that some people answered "maybe" in a lot of places (breaking record for @herbelin) but should be careful not to join more teams that they can actually handle review requests for. It's better to acknowledge that some teams are lacking manpower rather than trying to fill everything where help is missing and burning out. In the future, we might get a dashboard with statistics to better understand which components would need more help for pull request reviewing activities (not just based on team membership but also on actual reviewers' reactivity).

**Comment by gares** [comment]
>  It would be great if everyone could complete their profile with up-to-date info

I've pushed (maxime) quite a few times to have a "developers" webpage on the Coq website.

But I'm not so happy with the page containing stuff like PHD students (and their topics!), research interests (current works) and all sort of info that is doomed to become outdated. We already have our own homepages for this stuff. And we already have the credits section in the manual to know who did what.

If the page was on a website we could at least generate dynamically the list of components one maintains (that seems to be what that page should be used for).

To sum up *duplicating* info in a wiki page is a bad idea, sorry.

**Comment by Zimmi48** [comment]
>I've pushed (maxime) quite a few times to have a "developers" webpage on the Coq website (indeed I think it is relevant to the users to know who we are).

It has been a long time since we've talked about this. At least I've tried something. The point of this wiki page is not just to have a team page for outsiders, but also a way of sharing some broad info about current interests. This is information that is crucially lacking today, even among core developers. It is also interesting for contributors that are not complete outsiders but are also not fully aware of who is who.

>all sort of info that is doomed to become outdated

I am fully aware of this. That's the whole point of having put an "[up-to-date as of ????-??-??]" mention. As long as it is clear when the information is from, we can routinely go through the page and ask people who haven't updated it for a while to do it. If that were a webpage, someone would have to assume the task of maintaining it. The point of making this a wiki page is to share maintenance. But with the "up-to-date" mentions, at least we know how fresh the info is.

>We already have our own homepages for this stuff.

A webpage contains a lot of info, most of it might be totally irrelevant to Coq development, and it can be organized in many ways. The point of this page is to have a standard, short description format.

>If the page was on a website we could at least generate dynamically the list of components one maintains (that seems to be what that page should be used for).

Well sure, if you want to do this yourself, feel free to do it, it will probably be much better indeed. But if you don't do it yourself, I don't see this happening before many more years (that certainly wouldn't be a priority to me). My solution is "low-tech" but it can be available right now.

>To sum up duplicating info in a wiki page is a bad idea, sorry.

In many cases, it will provide info that is not available elsewhere. The rest of the time, it will provide info that is better structured.

Anyway, I almost regret mentioning this page here at all, because that wasn't the main point of this message. The main point was sorting out who should go in which team.

**Comment by Zimmi48** [comment]
As I feared, my long message has not been answered by anyone, so this will definitely need to be on the menu of the next Coq Call. In the meantime, feel free to answer anyway, especially for teams where adding more members is critically needed.

**Comment by ejgallego** [comment]
Hi @Zimmi48 , indeed I think people is a bit extra-busy at the beginning of the year so you may want to raise this in the Coq call too [or wait a bit more, but indeed as people is busy this will likely be lost on the sea of notifications]

> * @coq/extraction-maintainers has only two members (@pi8027 and @maximedenes). Should we add one of the persons that answered maybe (@ejgallego, @herbelin)?

I'm happy to help with extraction.

> * @coq/funind-maintainers has only two members (@Matafou, @forestjulien). Should we add new members among the ones that answered "maybe" (@ejgallego, @herbelin)?

I'm happy to help here too.

> * @coq/lib-maintainers has only one "yes" but three "maybe". Who should we add to the team among them (@SkySkimmer, @ejgallego, @herbelin)?

I think I'll leave other takes on this one for now.

> * @coq/library-maintainers has only two members: @maximedenes (yes) and @silene (previous code owner). Who should we add to the team among the four "maybe" (@SkySkimmer, @ejgallego, @ppedrot, @herbelin)?

> * @coq/vernac-maintainers has already three members (@mattam82, @maximedenes, @ejgallego) so might not need any of the two who answered "maybe".

These are interesting points; quite a few stuff in `library` has moved to `vernac`, and other componets will likely be moved too to the point of making it very thin; on the other hand `vernac` is getting quite large and we should split it soon in terms of generic stuff vs components.

> * @coq/test-suite-maintainers received only one "yes" (@SkySkimmer). Should we add the two that answered "maybe" (@ejgallego, @herbelin)?

I'd rather leave others take on this one for now.

> * @coq/toplevel-maintainers only has two members that answered "yes" (@gares, @ejgallego). Should they be joined by the only "maybe" (@herbelin)?

IMHO @herbelin is very active on the toplevel front so it makes sense for him to be added.

**Comment by SkySkimmer** [comment]
There are still
~~~
/tools/coq_tex*        @silene
# Secondary maintainer @gares
~~~
and
~~~
/test-suite/complexity/ @herbelin 
~~~
Are those intentional?

**Comment by Zimmi48** [comment]
Sort of, especially the first one: the intent was to clarify whether we wanted to keep this tool before creating a team dedicated to it.

**Comment by herbelin** [comment]
>     * @coq/doc-maintainers has already three people, but @cpitclaudel does not have so much time available, so it could make sense to have one more person join the team (the people that answered "maybe" are @anton-trunov and @herbelin).

I'm interested in interacting on the documentation, but I'm ok not being explicitly "responsible" for the good progresses of doc PRs, if @anton-trunov is willing to be the third one. (Note also that I'm clearly not an expert at sphinx syntax.)

>     * @coq/extraction-maintainers has only two members (@pi8027 and @maximedenes). Should we add one of the persons that answered maybe (@ejgallego, @herbelin)?

I answered "maybe" because I recently fixed an extraction bug and it was finally relatively easy, but I gladly let @ejgallego being the third.

>     * @coq/funind-maintainers has only two members (@Matafou, @forestjulien). Should we add new members among the ones that answered "maybe" (@ejgallego, @herbelin)?

I often struggled with funind, but I also gladly let @ejgallego being the third.

>     * @coq/lib-maintainers has only one "yes" but three "maybe". Who should we add to the team among them (@SkySkimmer, @ejgallego, @herbelin)?

I'm ok for being in then.

>     * @coq/library-maintainers has only two members: @maximedenes (yes) and @silene (previous code owner). Who should we add to the team among the four "maybe" (@SkySkimmer, @ejgallego, @ppedrot, @herbelin)?

I understand that @ejgallego has a focus on the structure of library and vernac and that libreay is going to thin. A no seems ok for me.

>     * @coq/ltac2-maintainers has only two members (@ppedrot, @MSoegtropIMC). Should we add someone from those who answered "maybe" (@herbelin, @JasonGross)?

I don't know ltac2 very well, so, if @JasonGross is ok, that may be more useful.

>     * @coq/stm-maintainers has only one member (@gares). Should we add the two that answered "maybe" (@maximedenes, @herbelin)?

My knowledge of the stm is very limited but OK to me.

>     * @coq/test-suite-maintainers received only one "yes" (@SkySkimmer). Should we add the two that answered "maybe" (@ejgallego, @herbelin)?

Ok to me.

>     * @coq/toplevel-maintainers only has two members that answered "yes" (@gares, @ejgallego). Should they be joined by the only "maybe" (@herbelin)?

Ok to me.

>     * @coq/typeclasses-maintainers has two members (@ppedrot and @mattam82, previous code owner). @ppedrot you did not answer "yes" nor "maybe" here, so do you wish to leave the team? Should we add @herbelin who answered "maybe" to it in any case?

I did not reach the level of really understanding the subtleties of typeclasses, but ok for a yes. Otherwise, what about @CohenCyril?
 
>     * @coq/universes-maintainers has only two members (@mattam82, @SkySkimmer). Does it need more members (@ppedrot and @herbelin answered "maybe")?

I believe that @ppedrot has a better overview of the implementation of universes than me.

>     * @coq/windows-build-maintainers has two members: @MSoegtropIMC and myself (maybe). Should we add any of the two other "maybe" (@gares, @herbelin)?

I'm a big ignorant of windows, but if @gares is not interested, it can be a yes as backup (or an acknowledgment of lacking manpower on this subject).

> In the future, we might get a dashboard with statistics to better understand which components would need more help for pull request reviewing activities (not just based on team membership but also on actual reviewers' reactivity).

That'd be great.

**Comment by Zimmi48** [comment]
@herbelin We kind of settled during a Coq Call on leaving things as they were and seeing if some teams needed more people based on experience. Of course, coming up with statistics based on actual data would be even better.

You can add yourself directly to any of the teams where you think you would be useful (while manually sending a notification to people in the team) but be careful not to take too much on your shoulders and not being able to follow up with all the review requests.

**Comment by herbelin** [comment]
> We kind of settled during a Coq Call on leaving things as they were

Ah, ok. No problem. By the way, no class nor meeting tomorrow, so, I'll be there.

> but be careful not to take too much on your shoulders and not being able to follow up with all the review requests.

I believe that this already the case... and not only me :smiley: I have no good solution though.

**Pull request merged**
